pirania-app/files/www/portal/js/voucher.js

```diff
@@ -35,98 +42,104 @@ function authVoucher () {
     id: 99,
     jsonrpc: '2.0',
     method: 'call',
-    params:[
+    params: [
       '00000000000000000000000000000000',
       'pirania',
       'auth_voucher',
       {
         voucher,
-        mac,
-      },
+        mac
+      }
     ]
   }
   fetch(url, {
     method: 'POST',
     body: JSON.stringify(authVoucherForm),
     headers: {
       'Access-Control-Allow-Origin': 'http://thisnode.info'
-    },
+    }
   })
-  .then(parseJSON)
-  .then((res) => {
-    hide(loader)
-    // debugger
-    if (res && res.result[1] && res.result[1].success) {
-      result.innerHTML = int[lang].success
-      show(result)
-      hide(errorElem)
-      init()
-      const urlTo = window.location.href.split('=')[1]
-      if (urlTo) {
-        window.location.href = `http://${urlTo}`
+    .then(parseJSON)
+    .then(res => {
+      hide(loader)
+      // debugger
+      if (res && res.result[1] && res.result[1].success) {
+        result.innerHTML = int[lang].success
+        show(result)
+        hide(errorElem)
+        init()
+        const urlTo = window.location.href.split('=')[1]
+        if (urlTo) {
+          window.location.href = `http://${urlTo}`
+        }
+      } else if (res && res.result[1] && !res.result[1].success) {
+        errorElem.innerHTML = int[lang].invalid
+        show(errorElem)
+      } else if (res.error) {
+        console.log(res.error)
+        errorElem.innerHTML = res.error
+        show(errorElem)
+
+        ubusError = true
       }
-    } else if (res && res.result[1] && !res.result[1].success) {
-      errorElem.innerHTML = int[lang].invalid
-      show(errorElem)
-    } else if (res.error) {
-      console.log(res.error)
-      errorElem.innerHTML = res.error
+      voucherElem.value = ''
+    })
+    .catch(err => {
+      console.log('Erro no Ubus', err)
+      errorElem.innerHTML = err
       show(errorElem)
-
       ubusError = true
-    }
-    voucherElem.value = ''
-  })
-  .catch((err) => {
-    console.log('Erro no Ubus', err)
-    errorElem.innerHTML = err
-    show(errorElem)
-    ubusError = true
-  })
-
+    })
```

Comment:
for some reason this block is a little entangled in the viewer, ... can't tell easier what this part of the change is about.
as a suggestion, it easens the review if the changes are organized in commits that tell what the change is about... it makes the reviewer focus on the relation between the commit comment and the commit content, and any issue that could arrise from there...

Reply:
Maybe my linter?